### ISSUE TITLE
Problem when starting

### ISSUE THREAD

**Issue by helionmelion** [flathub/io.kopia.KopiaUI#9]
Hello, I would like to report this issue. First time Im trying Kopia. Using archlinux, sway window manager (wayland, not X). A blank window open but the error is:

/opt/KopiaUI/kopia-ui
08:09:33.568 › failed to load
libva error: vaGetDriverNameByIndex() failed with unknown libva error, driver_name = (null)
[2968644:0614/080933.813984:ERROR:sandbox_linux.cc(377)] InitializeSandbox() called with multiple threads in process gpu-process.
08:09:34.073 › failed to load
08:09:34.575 › failed to load
08:09:35.076 › failed to load
08:09:35.577 › failed to load
08:09:36.079 › failed to load
08:09:36.581 › failed to load

"failed to load"... Until the process is killed.

**Comment by darkdragon-001** [comment]
Has this problem been resolved with a newer version of Kopia (and Electron)?

**Comment by jpds** [comment]
This issue hasn't been resolved as of `0.13.0` - the Electron app runs fine the first time it's run, but then if I close it (it continues to run in the background) and attempt to run it again - it briefly pops up before closing again.

All that's shown in a terminal window is:

```
❯ flatpak run io.kopia.KopiaUI
[2:0707/131242.540462:ERROR:bus.cc(399)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
Gtk-Message: 13:12:42.867: Failed to load module "canberra-gtk-module"
Gtk-Message: 13:12:42.868: Failed to load module "canberra-gtk-module"
[2:0707/131242.894876:ERROR:bus.cc(399)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
[2:0707/131242.894961:ERROR:bus.cc(399)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
13:12:42.953 › failed to load
13:12:42.957 › APPIMAGE env is not defined, current application is not an AppImage
```

I have to `flatpak kill` it before I can then run it normally again.

**Comment by darkdragon-001** [comment]
Probably best to report it upstream (https://github.com/kopia/kopia) as I have no idea what Kopia is doing (or supposed to do) on shutdown.
